src/evaluations/analyzer.py

```diff
@@ -456,6 +458,7 @@ def plot_one_estimator_under_one_scenario(df):
           w=self.plot_params[BARPLOT_SIZE_WIDTH_INCH],
           h=self.plot_params[BARPLOT_SIZE_HEIGHT_INCH])
       fig.savefig(plot_file)
+      plt.close(fig)
```

Comment:
I'd recommend using a context manager for this.

Reply:
good idea

Reply:
Per our offline discussion, I will leave this as is.